### ISSUE TITLE
Tauri build time very slow and generates very large files

### ISSUE THREAD

**Issue by Anderneku** [tauri-apps/tauri#7730]
DISCLAIMER: I'm quite new to Tauri and the problem at hand could likely be caused by my own ignorance.

I've been experimenting a lot with Tauri and been building a lot of apps and I've noticed that the build times are getting progressively slower and that the file size of the project is getting way too big. Like 3GB big.

I decided to do some digging to see what the cause is and upon inspection, I found that the biggest contributing file to the big file size was the **src-tauri** folder. I went further in and found out that the biggest file in the folder was the **target** folder.
NOTE: This is only when I had just created a Tauri project with the "npm create tauri-app@latest" command and hadn't touched anything.

I then went into the "debug" folder in the "target" folder as that had the bigger file size and thought a lot of all the folders in this folder had big file sizes (ranging from 100MB to 500MB), the biggest was the "deps" folder containing 2GB on it's own.

REMEMBER: I hadn't touched anything. I had only just created the tauri app and ran the "npm run tauri dev" command.

I would appreciate any help and response that comes with this
Thnx

**Comment by FabianLars** [comment]
tldr: sounds like it works as expected.

This is the major trade-off we have to live with for choosing Rust. We get small app sizes and good runtime performance but have to deal with much bigger build folders and slow build times :/

There really isn't much we can do to get major improvements here, only small things like removing dependencies and compiler improvements (talking about the rust project with the latter, not tauri itself)

**Comment by Anderneku** [comment]
Thanks for the help!
